efi/secureboot_policy.go

```diff
@@ -509,21 +501,29 @@ func (b *secureBootPolicyGenBranch) processPreOSEvents(events []*tcglog.Event, s
 		switch {
 		case e.PCRIndex < secureBootPCR && e.EventType == tcglog.EventTypeSeparator:
 			osPresent = true
-		case isKEKMeasurementEvent(e):
-			if err := b.processKEKMeasurementEvent(sigDbUpdates, sigDbUpdateQuirkMode); err != nil {
-				return xerrors.Errorf("cannot process KEK measurement event: %w", err)
-			}
 		case isDbMeasurementEvent(e):
+			// This is the db variable - requires special handling because it updates context
+			// for this branch.
 			if err := b.processDbMeasurementEvent(sigDbUpdates, sigDbUpdateQuirkMode); err != nil {
 				return xerrors.Errorf("cannot process db measurement event: %w", err)
 			}
-		case isDbxMeasurementEvent(e):
-			if err := b.processDbxMeasurementEvent(sigDbUpdates, sigDbUpdateQuirkMode); err != nil {
-				return xerrors.Errorf("cannot process dbx measurement event: %w", err)
+		case isSignatureDatabaseMeasurementEvent(e):
```

Comment:
if I understand the code change correctly the `data.VariableName == efi.GlobalVariable && data.UnicodeName == pkName` case of this before would have been handled below under `case e.PCRIndex == secureBootPCR` (now ``isSecureBootEvent(e)) ?

Reply:
That's correct.